docs/content/docs/usage/intl.md

```diff
@@ -5,7 +5,7 @@ title: Internationalize your content
 You have multiple solutions available to help you translate your user interface
 and handle multiple locales.
 
-Here we will explain 2 solutions based on
+Here we will the solution based on
```

Comment:
We also might explain in the top of this doc section that the idea is to have the language in the URL.